tests/integration_tests/README.md

```diff
@@ -1,52 +1,101 @@
+   * `mysql` (the MySQL cli client,
+     currently [mysql client 8.0 is not supported](https://github.com/pingcap/tidb/issues/14021))
+
+3. The user used to execute the tests must have permission to create the folder /tmp/tidb_cdc_test. All test artifacts
+   will be written into this folder.
+
+### Run integration tests in docker
+
+The following programs must be installed:
 
-3. The user used to execute the tests must have permission to create the folder /tmp/tidb_cdc_test. All test artifacts will be written into this folder.
+* [docker](https://www.docker.com/)
+* [docker-compose](https://docs.docker.com/compose/)
 
+We recommend that you provide docker with at least 6+ cores and 8G+ memory. Of course, the more resources, the better.
 
 ## Running
 
 ### Unit Test
 
-1. Unit test does not need any dependencies, just running `make unit_test` in root dir of source code, or cd into directory of a test case and run single case via `GO111MODULE=on go test -check.f TestXXX`.
+1. Unit test does not need any dependencies, just running `make unit_test` in root dir of source code, or cd into
+   directory of a test case and run single case via `GO111MODULE=on go test -check.f TestXXX`.
 
 ### Integration Test
 
+#### Run integration tests locally
+
 1. Run `make integration_test_build` to generate TiCDC related binaries for integration test
 
 2. Run `make integration_test` to execute the integration tests. This command will
 
    1. Check that all required executables exist.
    2. Execute `tests/integration_tests/run.sh`
 
-    > If want to run one integration test case only, just pass the CASE parameter, such as `make integration_test CASE=simple`.
+   > If want to run one integration test case only, just pass the CASE parameter, such as `make integration_test CASE=simple`.
 
-    > There exists some environment variables that you can set by yourself, variable details can be found in [test_prepare](_utils/test_prepare).
+   > There exists some environment variables that you can set by yourself, variable details can be found in [test_prepare](_utils/test_prepare).
 
-    > `MySQL sink` will be used by default, if you want to test `Kafka sink`, please run with `make integration_test_kafka CASE=simple`.
+   > `MySQL sink` will be used by default, if you want to test `Kafka sink`, please run with `make integration_test_kafka CASE=simple`.
 
 3. After executing the tests, run `make coverage` to get a coverage report at `/tmp/tidb_cdc_test/all_cov.html`.
 
+#### Run integration tests in docker
+
+> **Warning:**
+> These scripts and files may not work under the arm architecture,
+> and we have not tested against it.
+> Also, we currently use the PingCAP intranet address in our download scripts,
+> so if you do not have access to the PingCAP intranet you will not be able to use these scripts.
+> We will try to resolve these issues as soon as possible.
+
+1. If you want to run kafka tests,
```

Comment:
Do we still need zookeeper to test Kafka here?

Reply:
It is already in docker-compose. At the moment kraft is not production-ready, so we still need it.